csrankings-s.csv

```diff
@@ -264,6 +264,7 @@ Sang-Wook Kim,Hanyang University,http://www.agape.hanyang.ac.kr/members/prof,ed2
 Sang-Yoon Chang,UCCS,https://academics.uccs.edu/~schang2,m9j4C0gAAAAJ
 Sang-goo Lee,Seoul National University,http://ids.snu.ac.kr/site/members/M_Sang-goo_Lee.html,sg7xgbsAAAAJ
 Sang-hyo Park,Kyungpook National University,http://sites.google.com/view/knuvi,ZG8REuYAAAAJ
+Sang-Won Lee 0001,Sungkyunkwan University,http://prof.icc.skku.ac.kr/~swlee/,9a-vkpgAAAAJ

```

Comment:
Also needs alias entry for "Sang Won Lee 0001".

Reply:
I have added an alias.